assets/src/components/vehiclePropertiesCard.tsx

```diff
@@ -0,0 +1,182 @@
+import React, {
+  ComponentPropsWithoutRef,
+  MouseEventHandler,
+  ReactNode,
+  useId,
+} from "react"
+import { useCurrentTimeSeconds } from "../hooks/useCurrentTime"
+import { useNearestIntersection } from "../hooks/useNearestIntersection"
+import { Vehicle } from "../realtime"
+import { CloseButton } from "./closeButton"
+import StreetViewButton, { WorldPositionBearing } from "./streetViewButton"
+import { VehicleRouteSummary } from "./vehicleRouteSummary"
+
+interface VehicleProp {
+  vehicle: Vehicle
+}
+
+// #region Card Title Bar
+const DataStaleTime = ({
+  timestamp,
+}: {
+  timestamp: number
+}): React.ReactElement => {
+  const epochNowInSeconds = useCurrentTimeSeconds()
+  return (
+    <output
+      aria-label="Time Since Last Update Received"
+      className="data-stale-time label font-xs-reg"
+    >
+      Updated {epochNowInSeconds - timestamp} sec ago
+    </output>
+  )
+}
+
+const VehicleDataStaleTime = ({ vehicle }: VehicleProp): React.ReactElement => (
+  <DataStaleTime timestamp={vehicle.timestamp} />
+)
+// #endregion
+
+// #region Vehicle Work Info
+// Table Row Name Value Pair Data
+interface TrNameValueProps {
+  name: string
+  children: ReactNode
+  idPrefix?: string
+}
+
+// Table Row Name Value Pair
+const TrNameValue = ({
+  name,
+  children: value,
+  idPrefix,
+}: TrNameValueProps): React.ReactElement => {
+  const id = (idPrefix ?? name) + useId()
+  return (
+    <tr>
+      <th className="kv-key font-s-semi" scope="row" id={id}>
+        {name}
+      </th>
+      <td className="kv-value font-s-reg" aria-labelledby={id}>
+        {value}
+      </td>
+    </tr>
+  )
+}
+
+const VehicleWorkInfo = ({ vehicle }: VehicleProp): React.ReactElement => (
+  <>
+    <table className="m-vehicle-work-info">
+      <tbody className="m-vehicle-work-info__items">
+        <TrNameValue name="run">{vehicle.runId ?? "N/A"}</TrNameValue>
+        <TrNameValue name="vehicle">{vehicle.label ?? "N/A"}</TrNameValue>
+        <TrNameValue name="operator">
+          {[
+            vehicle.operatorFirstName,
+            vehicle.operatorLastName,
+            vehicle.operatorId ? `#${vehicle.operatorId}` : null,
+          ]
+            .filter((e) => e !== null)
+            .join(" ") || "N/A"}
+        </TrNameValue>
```

Comment:
**todo:** make sure that Fullstory is set to mask this value (@KaylaBrady set that up and may know more about how to do this).

Reply:
adding a `fs-mask` class should do the trick - that is what I used in https://github.com/mbta/skate/pull/1831

Reply:
Thanks for catching this!

Reply:
Is there any documentation on this?

I'm trying to figure out if I should put the `fs-mask` on the table cell that contains the sensitive value or on the table row itself

Reply:
@KaylaBrady correct me if I'm wrong, but I think it masks the element with the `fs-mask` class and all of its children, right? So either should be fine.